Kitodo-Query-URL-Import/src/main/java/org/kitodo/queryurlimport/QueryURLImport.java

```diff
@@ -434,7 +439,7 @@ private String createSearchFieldString(LinkedHashMap<String, String> searchField
                 .collect(Collectors.toList());
         String searchString = String.join(" AND ", searchOperands);
         if (Objects.nonNull(interfaceType) && SearchInterfaceType.SRU.equals(interfaceType)) {
-            return URLEncoder.encode(searchString, StandardCharsets.UTF_8.displayName());
+            return encodeQueryParameter(searchString, encoding);
```

Comment:
`encodeQueryParameter()` handles the `UnsupportedEncodingException`, please remove `throws UnsupportedEncodingException` from the method signature.